pyproject.toml

```diff
@@ -23,6 +23,7 @@ opentelemetry-distro = "^0.47b0"
 opentelemetry-instrumentation-fastapi = "^0.47b0"
 opentelemetry-exporter-prometheus = "^0.47b0"
 aiocache = "^0.12.2"
+httpx = "^0.27.0"
```

Comment:
Why was this moved?  Not seeing it imported anywhere in the non-tests files.

Reply:
It is used in `swdesc.py` (it was already here once - and I moved it back to test dependencies as I also did not see where it was used).